feat: add hint to REST-API (link to repo and meggsimums swagger export) (#10)
geostyler/geostyler-homepage: content/_index/docs.md

+++
fragment = "item"
date = "2020-10-06"
weight = 110
background = "secondary"
align = "right"

> title = "Geostyler: Read the docs"

Reviewer: ```suggestion
title = "GeoStyler: Read the docs"
```

> subtitle = "Read our API docs for details on how to use the Geostyler React-library to build flexible and powerfull GUI for styling maps in the browser"

Reviewer 2: ```suggestion
subtitle = "Read our API docs for details on how to use the Geostyler React-library to build flexible and powerful GUIs for styling maps in the browser"
```

> subtitle = "Read our API docs for details on how to use the Geostyler React-library to build flexible and powerful GUIs for styling maps in the browser"

Reviewer 2: ```suggestion
subtitle = "Read our API docs for details on how to use the GeoStyler React-library to build flexible and powerful GUIs for styling maps in the browser"
```
